OneCommunityGlobal/HighestGoodNetworkApp
# Zubing add GoogleDoc&Dropbox links on the profile

# Description
PRIORITY (URGENT) Jae: Add “Admin Document” and “Link to Media Files” to Profile page (WIP Zubing)
When creating a new user (Other Links → User Management → Create New User), the options exist to add an “Admin Document” and a “Link to Media Files”. 
If you add them during setup, they get added there. However, if you forget or they change, there’s no easy way to edit them. I’d like a permanent link above to the easy place to edit these because every person has them. 

## Related PRS (if any):
This frontend PR is related to frontend PR #1001 :

1. It fixed "add" and "update" button for the links.

## Main changes explained:
- Update UserLinks.jsx for including Dropbox link
- Update EditLinkModal.jsx for making names(Google Doc and Dropbox Folder) fixed and links fetched ti corresponding place.
- Update weeklySummaries.js for updating links to Weekly Summaries Reports page.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as owner user
4. go to Other Links→ User Management→ Create New User
5. create users: add “Admin Document” and “Link to Media Files” / add only one of them/ add none of them
6. go to these users' Profile→ Links Edit
7. verify:
there are name of Google Doc and Dropbox Folder fixed and corresponding box to add links.
for users created, if links were added when creating, check if the links are in the box and the names show on the profile under "Link Edit".
for users created, if links were not added when creating, check if the boxes are empty and there is no name show on the profile under "Link Edit".
delete the links and click" update, save changes" , the names on the profile under "Link Edit" would disappear.
Testing video:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92126559/89e7778d-d176-4e70-a710-8b6afa61030c

8. go to Reports→ Weekly Summaries Reports
9. Verify for Google Doc link:
the added link are updated to the Weekly Summary Reports page.
if modify the link in profile, this link in the Weekly Summary Reports page are changed correspondingly.
if delete the link, it shows "not present" in the Weekly Summary Reports page when clicking the Google Icon.
Testing video:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92126559/3d2eb9fb-0fa1-40fc-a91b-e9424fe36360

10. Verify for Media Folder(Dropbox link):
the added link are updated to the link box when submitting weekly summary.
if modify the link in profile, this link in the box when submitting weekly summary are changed correspondingly.
after submitting the summary, the link shows in the Weekly Summary Reports correctly.
Testing video:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92126559/193f92e4-1036-401e-b474-af65c0ef147d
